quilt/tools/package.py

```diff
@@ -257,7 +261,8 @@ def get_obj(self, node):
             self._check_hashes(node.hashes)
             return self._dataframe(node.hashes, node.format)
         elif isinstance(node, GroupNode):
-            hash_list = [h for c in node.preorder_tablenodes() for h in c.hashes]
+            hash_list = [h for c in node.preorder() if isinstance(c, TableNode)
+                         for h in c.hashes]
```

Comment:
avoid one letter varnames... e.g. what's 'c' ?

pylint should also complain - and if not, let's fix this ;-) for most cases (i for index var is fine...)

Reply:
Done. "C" stood for child, but it's not spelled out.